src/main/resources/logback.xml

```diff
@@ -95,6 +94,11 @@
   <!-- Suppress info logs from azure packages -->
   <logger name="com.azure" level="warn" />
 
+  <!--Log errors in GraphQL data fetchers. This is disabled by default.
+     Set it to "warn" to log these errors.
+  -->
```

Comment:
I don´t understand why this is set to `off` and the comment. Are you trying to mute this class since you are extending/overriding the logging? Error is ambigous in the context, it can mean "any problem" or just events with level=ERROR.

Reply:
I think I got it - it is ok if we change "error" to "issue".

Reply:
This is set to off by default for backward compatibility. Previously you would get these error messages only if you purposely switched on the `notprivacysafe.graphql` logger.

Reply:
"error" is the actual GraphQL term for this, as reported in the response, isn't it?